float32 angle
int8 class_id
int8 octave
int32[2] pt
int32 response
int32 size
